src/com/sproutlife/model/GameModel.java

```diff
@@ -15,102 +15,190 @@
 import com.sproutlife.model.step.GameStep;
 import com.sproutlife.model.step.GameStepListener;
 
+/**
+ * This is a class used to manage the game model essentials.
+ * 
+ * @author Alex Shapiro
+ */
 public class GameModel {
 
-    Echosystem echosystem;  
-
-    GameClock clock;
-
-    GameStep gameStep;
-
-    GameThread gameThread;
-
-    Settings settings;
-
-    Stats stats;   
-    
-    public GameModel(Settings settings, ReentrantReadWriteLock interactionLock) {
-        this.settings = settings;
-        this.clock = new GameClock();
-        echosystem = new Echosystem(clock);
-        gameStep = new GameStep(this);       
-        gameThread = new GameThread(this, interactionLock);
-        stats = new Stats(this);
-    }
-
-    public void performGameStep() {
-        incrementTime();
-        gameStep.perform();
-    }
-    /*
-    public GameStep getGameStep() {
-        return gameStep;
-    }
-    */
-    public Echosystem getEchosystem() {
-        return echosystem;
-    }
-
-    public Board getBoard() {
-        return echosystem.getBoard();
-    }          
-
-    public int getTime() {
-        return clock.getTime();
-    }
-
-    public GameClock getClock() {
-        return clock;
-    }
-
-    private void incrementTime() {
-        clock.increment();
-    }
-
-    public void resetGame() {
-        getEchosystem().resetCells();        
-        getEchosystem().pruneEmptyOrganisms();
-        getEchosystem().clearRetiredOrgs();
-        getStats().reset();
-        getClock().reset();
-        
-    }
-    
-    public Stats getStats() {
-        return stats;
-    }   
-
-    public Settings getSettings() {
-        return settings;
-    }
-    
-
-    public void setPlayGame(boolean playGame) {
-        if (playGame) {          
-            gameThread.setPlayGame(true);
-
-        } else {
-            gameThread.setPlayGame(false);            
-        }
-    }
-    
-    public GameThread getGameThread() {
-        return gameThread;
-    }
-    
-    public boolean getPlayGame() {
-        return gameThread.getPlayGame();
-    }
-
-    public void setGameStepListener(GameStepListener l) {
-        if (gameThread!=null) {
-            gameThread.setGameStepListener(l);
-            gameStep.setGameStepListener(l);
-        }
-    }
-
-    public void set(String s, Object o) {
-        getSettings().set(s,o);
-    }
+	/**
+	 * Calls Echosystem default constructor.
```

Comment:
These are not functions, so they don't "call" anything. I don't think these private members need comments (or you'd basically just be copying the class descriptions, which feels redundant).

Reply:
Whenever you create a new class member, a constructor is called. When it comes down to the redundant comments i agree, some of them aren't useful, but they are still documentation, nonetheless I will try to fix them, thanks for the feedback!

Reply:
These are initially null values right? So I don't see where a constructor comes in.